src/api/app/views/webui2/webui/cloud/ec2/configurations/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,8 @@
+= render partial: 'webui/main/breadcrumb_items'
+
+%li.breadcrumb-item
+  = link_to 'Cloud Upload', cloud_upload_index_path
+%li.breadcrumb-item
+  = link_to 'Cloud Configuration', cloud_configuration_index_path
+%li.breadcrumb-item.active
```

Comment:
I just realized, we're missing `{ 'aria-current' => 'page' }` for accessibility.

Reply:
I'll send a PR for this.